$NetBSD: patch-ogr_ogrsf__frmts_geojson_libjson_GNUmakefile,v 1.3 2019/03/09 01:54:59 gdt Exp $

Raise _XOPEN_SOURCE to 600 (for SunOS).
\todo Explain why this is necessary/correct.
\todo File ticket and get this fixed upstream.es

--- ogr/ogrsf_frmts/geojson/libjson/GNUmakefile.orig	2017-11-20 10:45:35.000000000 +0000
+++ ogr/ogrsf_frmts/geojson/libjson/GNUmakefile
@@ -21,8 +21,8 @@ CPPFLAGS := $(CPPFLAGS)
 
 default:	$(O_OBJ:.o=.$(OBJ_EXT))
 
-# -D_XOPEN_SOURCE=500 to enable strdup() definition in C11 mode
-CPPFLAGS := -D_XOPEN_SOURCE=500 $(CPPFLAGS)
+# -D_XOPEN_SOURCE=600 to enable strdup() definition in C11 mode
+CPPFLAGS := -D_XOPEN_SOURCE=600 $(CPPFLAGS)
 
 ../../o/%.$(OBJ_EXT):	%.c
 	$(CC) $(CFLAGS) $(CPPFLAGS) -c -o $@ $<
